sklearn/metrics/_classification.py

```diff
@@ -1201,10 +1201,9 @@ def fbeta_score(
     The F-beta score is the weighted harmonic mean of precision and recall,
     reaching its optimal value at 1 and its worst value at 0.
 
-    The `beta` parameter determines the weight of recall in the combined
-    score. ``beta < 1`` lends more weight to precision, while ``beta > 1``
-    favors recall (``beta -> 0`` considers only precision, ``beta -> +inf``
-    only recall).
+    The `beta` parameter represents the ratio of recall importance to
+    precision importance. For example, ``beta = 2`` makes recall twice as
+    important as precision, while ``beta = 0.5`` does the opposite.
```

Comment:
I think the original sentence about being above one and under one is still useful to have. Here is my suggestion:

```suggestion
    The `beta` parameter represents the ratio of recall to
    precision. `beta > 1` gives more weight to recall, while
    `beta < 1` favors precision. For example, `beta = 2` makes recall
    twice as important as precision, while `beta = 0.5` does the opposite.
```

Reply:
Hi @thomasjpfan, sure, let's keep that one : )
Updated